src/session.c

```diff
@@ -424,3 +439,9 @@ int jump_p2_queue(char* value)
 
 	return ret_val;
 }
+
+static void make_session_signature(char* out)
```

Comment:
if you put this before the first calling site, you dont need to declare it in session.h or anywhere else

Reply:
Modified